File: pkg/doctor/testdata/pi.php
<?php
$stripe = new \Stripe\StripeClient('sk_test_x');
// payment_method_types in a comment must NOT match
$stripe->paymentIntents->create([
  'amount' => 1099,
  'currency' => 'eur',
  'payment_method_types' => ['card', 'ideal'],
]);

File: pkg/doctor/testdata/Pi.java
import com.stripe.param.PaymentIntentCreateParams;

class Demo {
  void go() {
    // payment_method_types in a comment must NOT match
    PaymentIntentCreateParams params = PaymentIntentCreateParams.builder()
        .setAmount(1099L)
        .setCurrency("eur")
        .addPaymentMethodType("card")
        .build();
  }
}
